features/situation-feedback/elastic/src/main/resources/feedback-template.json

```diff
@@ -63,6 +63,12 @@
                     "type": "text",
                     "norms": false
                 },
+                "root_cause": {
+                    "type": "boolean"
+                },
+                "tags": {
+                    "type": "keyword"
```

Comment:
Just checking, does this actually get mapped correctly to the collection? Or will it only capture one element?

Reply:
it's correctly mapped